lab-materials/03/06/llm_usage.py

```diff
@@ -24,11 +22,11 @@ def infer_with_template(input_text, template):
         verbose=False,
     )
     
-    PROMPT = PromptTemplate.from_template(template)
+    prompt = PromptTemplate(input_variables=["car_claim"], template=template)
```

Comment:
@RHRolun why we're selecting only car_claim? will be only targeting this input_variable?

Reply:
Correct, the templates only have a single variable in them and it's "car_claim".